### PR TITLE
Documentation Update

### PR DESCRIPTION
**Pull request by ghost**
## The following commits were made #59 :

+ Removed bad image, fixed existing typos and updated markdown on [code-of-conduct.md](https://github.com/dwyl/summer/blob/master/code-of-conduct.md).

+ Updated the [freelance-agreements/readme.md](https://github.com/dwyl/summer/blob/master/freelance-agreements/readme.md) markdown and minor grammar changes.

+ Updated the [freelance-agreements/freelance-agreement.md](https://github.com/dwyl/summer/blob/master/freelance-agreements/freelance-agreement.md) markdown code.

+ Updated markdown code of [invoicing-dwyl.md](https://github.com/dwyl/summer/blob/master/invoicing-dwyl.md), [summer-2015-readme.md](https://github.com/dwyl/summer/blob/master/summer-2015-readme.md) and [useful-info.md](https://github.com/dwyl/summer/blob/master/useful-info.md).

### CODE REVIEW

**Comment by ghost** [comment]
Since I made multiple commits, I hope this facilitates your job of reviewing the changes before merging the PR. I accidentally clicked the wrong button on the other PR, sorry for any confusion it may have caused.

File: code-of-conduct.md [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 
 0. The [***Golden Rule***](https://en.wikipedia.org/wiki/Golden_Rule): treat others as one would like others to treat oneself.
 
-1. **Be happy.** We're surrounding ourselves with sun, sea/forest, fresh/healthy food and great people so good vibes are mandatory (read: almost unavoidable).
+1. **Be happy.** We're surrounding ourselves with the sun, sea/forest, fresh/healthy food and great people so good vibes are mandatory (read: almost unavoidable).
 
 2. **Be respectful of other people's working hours.** It's an awesome place but we're here to work as well as relax and enjoy the beautiful surroundings. People may be working when you're relaxing and vice versa so keep that in mind!
 
-3. **Be tidy.** Common sense/courtesy apply here - always clean any mess on counters/floors, wash dishes/cultery you use and leave things as you find them (or better!).
+3. **Be tidy.** Common sense/courtesy apply here - always clean any mess on counters/floors, wash dishes/cutlery you use and leave things as you find them (or better!).
```
Comment (nelsonic) [inline code-of-conduct.md:9]: Good catch. Thanks for fixing. 👍🏻

**Review by nelsonic — APPROVED** [review]
@syspentarc thanks very much for making all these updates & improvements! 🙌☀️

**Comment by nelsonic** [comment]
Once you feel the Pull Request is done, please hit the “Ready for Review” button and we will merge it. 👍🏻

**Comment by ghost** [comment]
I just made it “Ready for Review".

**Comment by nelsonic** [comment]
@syspentarc thanks again! 🙌

**Pull request merged**